packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormAccountSelection/TransferFormAccountSelection.scss

```diff
@@ -0,0 +1,126 @@
+.wallets-transfer-form-account-selection {
+    display: flex;
+    width: 40rem;
+    flex-direction: column;
+    align-items: flex-start;
+    border-radius: 0.8rem;
+    background: var(--system-light-8-primary-background, #fff);
+    /* Shadows/xxxl */
+    box-shadow: 0rem 3.2rem 6.4rem 0rem rgba(14, 14, 14, 0.14);
+    max-height: 52.8rem;
+
+    @include mobile {
+        position: absolute;
+        inset: 0;
+        width: 100%;
+        height: 100%;
```

Comment:
hey @rostislav-deriv 
i think so, but in my case i need to add width because of some css properties (which is not in this file) affect on this class.